R/render.R

```diff
@@ -343,8 +372,8 @@ add_links <- function(path, main_language = main_language,
     } else {
       path_rel(path, output_folder, path_language, main_language)
     }
-    href <- sprintf("%s/%s", site_url, new_path)
-    no_translated_version <- !fs::file_exists(file.path(output_folder, new_path))
+    href <- sprintf("%s/%s", site_url, new_path) # nolint: nonportable_path_linter
```

Comment:
Just when I think I've looked up all the linters there are, out comes a new one 😅